mivs/automated_emails.py

```diff
@@ -0,0 +1,14 @@
+from mivs import *
```

Comment:
We can configure automated emails on either a per-studio basis or a per-game basis.  In both cases these emails will go to the primary contact for the studio.